sdks/python/apache_beam/dataframe/pandas_docs_test.py

```diff
@@ -33,10 +39,15 @@
 PANDAS_DIR = os.path.expanduser("~/.apache_beam/cache/pandas-" + PANDAS_VERSION)
 PANDAS_DOCS_SOURCE = os.path.join(PANDAS_DIR, 'doc', 'source')
 
+parallelism = None
+
 
 def main():
-  # Not available for Python 2.
-  import urllib.request
+  parser = argparse.ArgumentParser()
+  parser.add_argument('-p', '--parallel', type=int, default=0)
```

Comment:
nit: add a help string indicating the default is 0, which will use the cpu count

Reply:
Done.